docs/pages/reference/resources.mdx

```diff
@@ -103,6 +103,32 @@ Roles govern access to databases, SSH servers, Kubernetes clusters, web services
 
 (!docs/pages/includes/role-spec.mdx!)
 
+### Role versions
+
+Roles v5 and v6 can only restrict actions on pods (like executing in them).
```

Comment:
This doc covers more than Kubernetes access, so I would elaborate a bit. If you're not using kube access, role v5 and v6 are effectively the same, right?

Reply:
I think that v5, v6 and v7 are equivalent if you don't use kube access. We backported all the other fields

Reply:
Edited in https://github.com/gravitational/teleport/pull/37360/commits/6b160c327822702493d1483d8fc13d2e1b39c276